roles/importer/files/importer/checkpointR8x/fwcommon.py

```diff
@@ -464,29 +464,47 @@ def get_objects(nativeConfig: dict, importState: ImportStateController) -> int:
             
     # loop over sub-managers in case of mds
     manager_index = 0
-    for managerDetails in manager_details_list:
-        cpManagerApiBaseUrl = importState.MgmDetails.buildFwApiString()
+    for manager_details in manager_details_list:
+        cp_api_url = importState.MgmDetails.buildFwApiString()
         
         # getting Original (NAT) object (both for networks and services)
-        sid = loginCp(managerDetails)
-        if managerDetails.IsSuperManager or len(manager_details_list) == 1:
-            origObj = cp_getter.getObjectDetailsFromApi(cp_const.original_obj_uid, sid=sid, apiurl=cpManagerApiBaseUrl)['chunks'][0]
-            anyObj = cp_getter.getObjectDetailsFromApi(cp_const.any_obj_uid, sid=sid, apiurl=cpManagerApiBaseUrl)['chunks'][0]
-            noneObj = cp_getter.getObjectDetailsFromApi(cp_const.none_obj_uid, sid=sid, apiurl=cpManagerApiBaseUrl)['chunks'][0]
-            internetObj = cp_getter.getObjectDetailsFromApi(cp_const.internet_obj_uid, sid=sid, apiurl=cpManagerApiBaseUrl)['chunks'][0]
+        sid = loginCp(manager_details)
+        if manager_details.IsSuperManager or len(manager_details_list) == 1:
+            origObj = cp_getter.getObjectDetailsFromApi(cp_const.original_obj_uid, sid=sid, apiurl=cp_api_url)['chunks'][0]
+            anyObj = cp_getter.getObjectDetailsFromApi(cp_const.any_obj_uid, sid=sid, apiurl=cp_api_url)['chunks'][0]
+            noneObj = cp_getter.getObjectDetailsFromApi(cp_const.none_obj_uid, sid=sid, apiurl=cp_api_url)['chunks'][0]
+            internetObj = cp_getter.getObjectDetailsFromApi(cp_const.internet_obj_uid, sid=sid, apiurl=cp_api_url)['chunks'][0]
 
         # get all objects
-        for obj_type in cp_const.api_obj_types:
-            object_table = get_objects_per_type(obj_type, show_params_objs, sid, cpManagerApiBaseUrl)
+        if manager_index==0:
```

Comment:
As we spoke, the behavior that first manager is supermanager needs explicitly be established or we need to solve this differently here

Reply:
replaced with 
        if manager_details.IsSuperManager or len(manager_details_list) == 1: